drivers/input/touchscreen/edt-ft5x06.c

```diff
@@ -274,7 +283,31 @@ static irqreturn_t edt_ft5x06_ts_isr(int irq, void *dev_id)
 			error = edt_ft5x06_ts_readwrite(tsdata->client,
 							sizeof(cmd), &cmd,
 							datalen, &rdbuf[offset]);
+			if (error) {
+				dev_err_ratelimited(dev,
+						    "Unable to fetch data, error: %d\n",
+						    error);
+				goto out;
+			}
+
+			/* When polling FT5x06 without IRQ: initial register
+			 * contents could be stale or undefined; ignore them
+			 * until something changes or num_points becomes 0.
+			 */
```

Comment:
As per the comment, now that you're delaying and don't see the awkward drop to 0 points, can't we now just filter until num_points to hits 0?

Reply:
Can probably simplify by detecting change to `num_points` only.